docs/GUIDE.md

```diff
@@ -0,0 +1,229 @@
+
+- Email
+- Password
+
+_OIDC Providers coming soon._
+
+### Password Retrieval
+
+Forgot your password? Thunderdome can send a link to your email.
+
+## Profile
+
+Warrior, it is all about you! Control your Thunderdome experience.
+
+### Details
+
+- Name
+This will be visible to others.
+- Email
+Update your account email, or enter one if you are a guest.
+- Country (optional)
+- Locale, default: English
+8 locales to choose from.
+- Company (optional)
+- Job Title (optional)
+- Theme, default: auto
+If you prefer a darker or lighter interface, you may choose it here.
+- Option: Enable Battle Notification, default: true
+- Avatar, default: robohash
+Several others to choose from, pick your flavor; mp, identicon, monsterid, wavatar, retro.
+
+### API Access
+
+Create an API key to integrate Thunderdome with your tools. 
+
+See API Documentation here [Thunderdome API Docs](https://thunderdome.dev/swagger/index.html)
+
+### Jira Integration (premium only)
+
+Integrate directly with your team's backlog.
+
+_Other integrations coming soon._ 
+
+### Delete Account
+
+This is the Thunderdome, but you are free to leave. We will erase all data about you.
+
+## Battle
+
+In Thunderdome, an agile poker planning session is known as a Battle.
+
+You can create a battle to determine the size of a story, or join one in progress.
+
+### Create a Battle
+
+- Name
+- Team (optional)
+- Point Range Allowed, default: [ 1, 2, 3, 5, 8, 13, ? ]
+- Plans
+Upload an XML or CSV for plans, or add manually. See note in Plans.
+- Point Average Rounding, default: Ceil
+Other options; Round, Floor.
+- Option: Auto Finish Voting, default: true
+- Option: Hide Voter Identity, default: false
+- Passcode (optional)
+- Leader Code (optional)
+
+### Battle
+
+The planning session is real-time, each warrior chooses the size for the story and votes are shown when everyone has finished.
+
+If the team agrees, the battle is over. If not, then it has just begun!
+
+![image](https://user-images.githubusercontent.com/846933/95778842-eb76ef00-0c96-11eb-99d8-af5d098c12ee.png)
+
+### Plans
+
+This can be a list of Stories, Bugs, Tasks, etc. and serves as a queue for team voting.
+
+#### Import plans from Jira Cloud
+
+Premium feature.
+
+#### Import plans from Jira XML
+
+Upload.
+
+#### Import plans from a CSV file
+
+The CSV file must include all the following fields with no header row:
+
+_Type,Title,ReferenceId,Link,Description,AcceptanceCriteria_
+
+#### Create a Plan
+
+- Type, default: Story
+Other Types: Bug, Spike, Epic, Task, Subtask
+- Name
+- Reference ID
+- Link
+- Priority
+Priorities; Blocker, Highest, High, Medium, Low, Lowest
+- Description
+A full text editor is supplied to provide a detailed description.
+- Acceptance Criteria
+A full text editor is supplied, feel free to use Gherkin statements.
+
+### Warriors
+
+See who is voting or become a spectator.
+
+### Invite
+
+Send a link for others to join the battle.
+
+## Retrospectives
+
+Review your battles.
```

Comment:
It's not a review of battle, but an agile sprint retrospective.

Reply:
```suggestion
Facilitates an agile sprint retrospective.
```